lib/tools/flv-inspector.js

```diff
@@ -1,13 +1,18 @@
-(function(window, muxjs) {
+'use strict';
 
 var
+  tagTypes = {
+    0x08: 'audio',
+    0x09: 'video',
+    0x12: 'metadata'
+  },
   hex = function (val) {
     return '0x' + ('00' + val.toString(16)).slice(-2).toUpperCase();
   },
   hexStringList = function (data) {
-    var arr = [];
-
-    while(data.byteLength >0) {
+    var arr = [], i;
+    /* jshint -W086 */
```

Comment:
The alternative to this would be annotating each of the cases with something like `/* falls through */`, right? That's a bit more explicit but not a big deal.